extensions/positron-assistant/package.json

```diff
@@ -135,7 +135,7 @@
           },
           "positron.assistant.streamingEdits.enable": {
             "type": "boolean",
-            "default": false,
+            "default": true,
             "description": "%configuration.streamingEdits.enable%",
             "tags": ["experimental"]
```

Comment:
Since this is enabled by default, should it still be tagged with `experimental`?

Reply:
Good point, fixed in 8d97d4b.